src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry-configuration/telemetry-configuration.component.html

```diff
@@ -0,0 +1,211 @@
+<cd-loading-panel *ngIf="loading && !error"
+                  i18n>Loading configuration...</cd-loading-panel>
+<cd-alert-panel type="error"
+                *ngIf="loading && error"
+                i18n>The configuration could not be loaded.</cd-alert-panel>
+
+<div class="cd-col-form"
+     *ngIf="!loading && !error">
+  <form name="form"
+        #formDir="ngForm"
+        [formGroup]="form"
+        novalidate>
+    <div class="card">
+      <div class="card-header"
+           i18n>Step 1 of 2: Telemetry report configuration</div>
+      <div class="card-body">
+        <div i18n>The telemetry module sends anonymous data about the cluster back to the Ceph developers
+          to help understand how Ceph is used and what problems users may be experiencing.<br/><br/>
+          The data being reported does <b>not</b> contain any sensitive data like pool names, object names, object contents,
+          hostnames, or device serial numbers. It contains counters and statistics on how the cluster has been
+          deployed, the version of Ceph, the distribution of the hosts and other parameters which help the project
+          to gain a better understanding of the  way Ceph is used. The data is sent over HTTPS to telemetry.ceph.com.
```

Comment:
This phrase includes a valid URI.
```suggestion
          to gain a better understanding of the  way Ceph is used. The data is sent secured to https://telemetry.ceph.com.
```